mods/ts/rules/aircraft.yaml

```diff
@@ -287,3 +287,44 @@ APACHE:
 	RenderSprites:
 	Hovers:
 
+HUNTER:
+	Inherits@1: ^GainsExperience
+	Inherits@2: ^ExistsInWorld
+	Valued:
+		Cost: 1000
+	Tooltip:
+		Name: Hunter-Seeker Droid
+	Health:
+		HP: 500
+	Armor:
+		Type: Light
+	DemoTruck:
+	Explodes:
+		Weapon: SuicideBomb
+		EmptyWeapon: SuicideBomb
+	Aircraft:
```

Comment:
The H-S doesn't reveal shroud, but can be moved into it. Don't know how the original handled it, but should it either have `RevealsShroud` or `MoveIntoShroud` set to false on `Aircraft`?